docs/userguides/compile.md

```diff
@@ -111,6 +111,23 @@ solidity = compilers.get_compiler("solidity", settings=settings["solidity"])
 vyper.compile([Path("path/to/contract.sol")])
 ```
 
+### Settings Artifacts
+
+Compiler build artifacts can be found in the the `<project>/.build/compilers.json` file.
```

Comment:
actually, not sure if this location makes sense, because the `Compiler` object in the manifest is not expected to be a link to another file.

can probably just be stored directly in the `<project>/.build/__local__.json` file

Reply:
I though we were going to keep everything separate so people to be choosier about the files they need to host and stuff? Didn't we chat about that recently on GitHub somewhere?

Reply:
I suppose the contract types being separate is probably ok enough for now.

Reply:
I'm realizing this is a much bigger refactor ... Dependencies heavily rely on the compiler data already and messing with __local__.json is giving me a chicken and egg problem right now. Closing for now.